dotnet/iwebapi/k8s/base/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ContainerRepositoryPlaceholder
+  labels:
+    app.openshift.io/runtime: dotnet
+spec:
+  replicas: 2
```

Comment:
consider adding `spec/revisionHistoryLimit: 0`

Reply:
creates a lot of noise in argo, and argo is used for rollbacks etc instead of kubernetes